app/presenters/menu/section.rb

```diff
@@ -84,5 +94,9 @@ def parent_path(acc = [])
       acc << id
       @parent.present? ? @parent.parent_path(acc) : acc
     end
+
+    def load_children?
```

Comment:
Why is this required? Is it used somewhere outside? If yes, why isn't set with `set_attribute`?